README.md

```diff
@@ -17,3 +17,11 @@ samples, guidance on mobile development, and a full API reference.
 
 ![SYLLOGO](https://user-images.githubusercontent.com/80233893/147422418-97904be3-ddf3-460e-821a-23b02fd665ed.png)
 
+## App Developement plan: 
+CODEOWNERS: 
```

Comment:
Maybe add names as well?

Reply:
That's a good idea, but I think shouldnn't